packages/cli/lib/lib/webpack/run-webpack.js

```diff
@@ -143,7 +143,7 @@ function writeJsonStats(cwd, stats) {
 function allFields(stats, field, fields = [], name = null) {
 	const info = stats.toJson({
 		errors: true,
-		warnings: false,
+		warnings: true,
```

Comment:
Noticed when debugging.

We attempt to log warnings if `stats.hasWarnings()` is true:

https://github.com/preactjs/preact-cli/blob/3f389c2486a3790ae9e18b67ac959f87de19c973/packages/cli/lib/lib/webpack/run-webpack.js#L167-L169

However, `info.warnings` will always be empty w/out this change.